src/RhinoInside.Revit.GH/Components/Element/SpatialElement/AddSpaceSeparatorLine.cs

```diff
@@ -107,12 +108,14 @@ protected override void TrySolveInstance(IGH_DataAccess DA)
       );
     }
 
-    bool Reuse(ARDB.ModelCurve spaceSeparator, ARDB.Document doc, Curve curve, ARDB.View view)
+    bool Reuse(ARDB.ModelCurve spaceSeparator, Curve curve, ARDB.View view)
     {
       if (spaceSeparator is null) return false;
       if (spaceSeparator.OwnerViewId != view.Id) return false;
       var projectedCurve = Curve.ProjectToPlane(curve, view.SketchPlane.GetPlane().ToPlane());
       if (!projectedCurve.ToCurve().IsAlmostEqualTo(spaceSeparator.GeometryCurve)) return false;
+
```

Comment:
returning false here prevents the curve is updated.

Reply:
Done!